src/doc/man/cargo-install.md

```diff
@@ -121,6 +121,13 @@ Filesystem path to local crate to install from.
 List all installed packages and their versions.
 {{/option}}
 
+{{#option "`-n`" "`--dry-run`" }}
+(unstable) Cut the install operation short and display what it *would* have
+usually done. The normal version check and crate downloads still occur exactly
+like without the option flag, but the compilation and the actual installation
```

Comment:
imo this over-specificies (I could see us changing compilation)

Our other dry-run help descriptions are pretty brief

publish

> Perform all checks without uploading.

add

> Don’t actually write the manifest

update

> Displays what would be updated, but doesn’t actually write the lockfile.

Reply:
Changed to something less specific